src/main/java/org/kiwiproject/test/dropwizard/app/PostgresAppTestExtension.java

```diff
@@ -0,0 +1,63 @@
+package org.kiwiproject.test.dropwizard.app;
+
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.zonky.test.db.postgres.embedded.LiquibasePreparer;
+import io.zonky.test.db.postgres.junit5.EmbeddedPostgresExtension;
+import io.zonky.test.db.postgres.junit5.PreparedDbExtension;
+import lombok.Getter;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * Multi-purpose extension for {@link Application} testing requiring a postgres database. The extension will spin up an
+ * embedded postgres database instance, run the migrations and then configure the {@link DropwizardAppExtension} with the
+ * database information.
+ * <p>
+ * To include this extension in an AppTest then add the following at the top of the class:
+ * <p>
+ * {@code @RegisterExtension}
+ * <br>
+ * {@code public static PostgresAppTestExtension<AppConfiguration> APP = new PostgresAppTestExtension("migrations.xml", "config.yml", App.class);}
+ * <p>
+ * The test instance will have access to the application and the postgres instance by calling {@code APP.getApp()} and {@code APP.getPostgres()}
```

Comment:
Probably ought to capitalize all `postgres` as `Postgres` right? Or should we be formal and call it "PostgreSQL" which is the real name?

Reply:
Done